### PR TITLE
rewritten the section on integration methods

### PR DESCRIPTION
**Pull request by wibraun**
- attempt to fix ticket:4246

### CODE REVIEW

File: UsersGuide/source/solving.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+and a more mathematical approach can be found in :cite:`Hairer:1993`.
+
+.. _dassl :
+
+DASSL
+~~~~~
+
+DASSL is the default solver in OpenModelica, because of a severals reasons.
+It is an implicit, higher order, multi-step solver with a step-size control
+and with this properties it is quite stable for a width range of models.
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: Should be `with these properties` or `with its properties` and `wide range`

File: UsersGuide/source/solving.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+-------------------
+
+By default OpenModelica transforms a Modelica model into an ODE
+representation to perform a simulation by using numerical integration
+methods. This section contains additional information about the different
+integration methods in OpenModelica. They can be selected by the method
+parameter of the :ref:`simulate` command or the :ref:`-s simflag <simflag-s>`.
+
+The different methods are also called solver and can be distinguished by
+there characteristic:
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: `their`

File: UsersGuide/source/solving.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+DASSL is the default solver in OpenModelica, because of a severals reasons.
+It is an implicit, higher order, multi-step solver with a step-size control
+and with this properties it is quite stable for a width range of models.
+Furthermore it has a mature source code, which was originally developed
+in the eighties an initial description may be found in :cite:`PetzoldDASSL:1982`.
+
+This solver is based on backward differentiation formula (BDF), which is
+a family of implicit methods for numerical integration. The used implementation
+is called DASPK2.0 (see [#f3]_) and itis translated automatically to C
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: space between `it is`

File: UsersGuide/source/solving.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+by f2c (see [#f4]_).
+
+The following simulation flags can be used to adjust the behavior of the
+solver for specific simualtion problems:
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: `simulation`

File: UsersGuide/source/solving.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+:ref:`idaMaxConvFails <simflag-idaMaxConvFails>`,
+:ref:`idaNonLinConvCoef <simflag-idaNonLinConvCoef>`,
+:ref:`idaMaxErrorTestFails <simflag-idaMaxErrorTestFails>`.
+
+Basic Explicit Solver
+~~~~~~~~~~~~~~~~~~~~~
+
+The basic explicit solver are performing with a fixed step-size and
+differs only in the integration order. The step-size is based on the
+numberOfIntervals, the startTime and stopTime parameters in the
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: singular or plural??? `solver / is / differs` or `solvers / are / differ`

File: UsersGuide/source/solving.rst [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+- euler - order 1
+- heun - order 2
+- rungekutta - order 4
+
+Basic Implicit Solver
+~~~~~~~~~~~~~~~~~~~~~
+
+The basic implicit solver are all based on the non-linear solver KINSOL
+from the SUNDIALS suite. The underlining linear solver can be modified
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: singuar or plural?

File: UsersGuide/source/solving.rst [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+- trapezoid - order 2
+- imprungekutta - Based on Radau IIA and Lobatto IIIA defined by its
+  Butcher tableau where the order can be adjusted by :ref:`-impRKorder <simflag-imprkorder>`.
+
+
+Experimental Solver
+~~~~~~~~~~~~~~~~~~~
+
+The following solver are marked as experimental, mostly because they
+are till now not tested very well.
```
Comment (hkiel) [inline UsersGuide/source/solving.rst]: plural? `solvers`

**Comment by wibraun** [comment]
@hkiel Thank you!

**Comment by adeas31** [comment]
Is this PR okay to merge now?

**Comment by wibraun** [comment]
@adeas31 From my side: yes.
Maybe it could receive some polishing on the language level, but it should do the job for now and it is better then before.

**Comment by casella** [comment]
Sounds good to me also. I posted some comments in the ticket.

**Pull request merged**